.github/workflows/components-to-doc.yml

```diff
@@ -0,0 +1,66 @@
+name: Components to Documents
+on:
+  push:
+    branches:
+      - "master"
+  # pull_request:
+    # branches:
+      # - "*"
+  release:
+    types: [published]
+
+jobs:
+  CopyComponents:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Make components to .md files
+        run: |
+            
+            function join_by {
+              local d=${1-} f=${2-}
+              if shift 2; then
+                echo "$f${@/#/$d}"
+                
+              fi
+            }
+            mkdir output
+            folder_path=templates/oam/workloads/*
```

Comment:
@Revolyssup when this filepath changes... let's be sure to include a global find and replace. // @gr455